Index: src/wp-includes/post.php
===================================================================
--- src/wp-includes/post.php	(revisão 381)
+++ src/wp-includes/post.php	(cópia de trabalho)
@@ -3594,7 +3594,7 @@
 	if ( empty($url) ) //If any of the above options failed, Fallback on the GUID as used pre-2.7, not recomended to rely upon this.
 		$url = get_the_guid( $post->ID );
 
-	if ( 'attachment' != $post->post_type || empty($url) )
+	if ( empty($url) )
 		return false;
 
 	return apply_filters( 'wp_get_attachment_url', $url, $post->ID );
